index.js

```diff
@@ -1,20 +1,20 @@
 'use strict';
 const net = require('net');
 
-const getPort = port => new Promise((resolve, reject) => {
+const getPort = (port, host) => new Promise((resolve, reject) => {
 	const server = net.createServer();
 
 	server.unref();
 	server.on('error', reject);
 
-	server.listen(port, () => {
+	server.listen({port, host}, () => {
```

Comment:
I have just double checked. It is not 100% clear to me, but it seems that `host` is mandatory with the object approach which is not what we're looking for here.